src/pages/dmt5/components/_side-tab.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
```

Comment:
how does this different with `components/elements/side-tab`, it used in `/pages/trade-types/options/_how-options-works.js`

Reply:
Yes, I thought about making them as one shared component. but, I think I have to put the new component in a shared folder. Right?